Resources/config.xml

```diff
@@ -122,6 +122,33 @@
             </options>
         </element>
 
+        <element type="button">
+            <name>btnPaymentUpdate</name>
```

Comment:
please add the position entry.
this is mandatory to always keep a consisent plugin view for new installations and updates